### ISSUE TITLE
Slide Transition - jump before transition

### ISSUE THREAD

**Issue by ealymbaev** [andkulikov/transitions-everywhere#17]
I have 3 views on the first scene and only one of them on the second scene.

I am using Slide transition to slide out the 2 views out of the screen. These 2 views are placed in Relative layout and positioned using "above" and "below" attributes.

**Scene 1**

```
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    >

    <TextView
        android:id="@+id/caption"
        style="@style/RegistrationTextView"
        android:text="@string/let_us_get_you_started"
        />

    <LinearLayout
        android:id="@+id/edit_text_container"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_below="@id/caption"
        android:layout_above="@+id/next"
        android:orientation="vertical"
        android:weightSum="2"
        android:animateLayoutChanges="true"
        >

        <LinearLayout
            android:layout_width="match_parent"
            android:layout_height="0dp"
            android:layout_weight="1"
            android:gravity="center|bottom"
            android:animateLayoutChanges="true"
            >

            <EditText
                android:id="@+id/edit_text"
                style="@style/RegistrationEditText"
                android:imeOptions="actionNext"
                android:hint="@string/email_address"
                />

            <EditText
                android:id="@+id/edit_text_surname"
                style="@style/RegistrationEditText"
                android:layout_marginLeft="10dp"
                android:hint="@string/surname"
                android:inputType="textNoSuggestions|textCapWords"
                android:imeOptions="actionNext"
                android:visibility="gone"
                />
        </LinearLayout>

        <TextView
            android:id="@+id/validation_message"
            android:layout_width="match_parent"
            android:layout_height="0dp"
            android:layout_weight="1"
            android:gravity="center"
            android:textColor="@color/validation_message"
            android:visibility="invisible"
            />
    </LinearLayout>

    <Button
        android:id="@+id/next"
        style="@style/RegistrationButton"
        android:text="@string/next"
        />

    <ProgressBar
        android:id="@+id/progress_bar"
        android:layout_width="30dp"
        android:layout_height="30dp"
        android:layout_alignParentBottom="true"
        android:layout_centerHorizontal="true"
        android:layout_marginBottom="20dp"
        android:visibility="gone"
        />

</RelativeLayout>
```

**Scene 2**

```
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    >

    <Button
        android:id="@+id/next"
        style="@style/RegistrationButton"
        android:text="@string/next"
        />

</RelativeLayout>
```

**Transition**

```
<transitionSet
    xmlns:app="http://schemas.android.com/apk/res-auto"
    app:transitionOrdering="sequential"

    >

    <slide app:slideEdge="left" app:duration="1000">
        <targets>
            <target app:targetId="@id/edit_text_container" />
        </targets>
    </slide>

    <transitionSet
        app:duration="3000"
        >

        <targets>
            <target app:targetId="@id/caption" />
        </targets>

        <slide app:slideEdge="left" />
        <fade app:fadingMode="fade_out" />

    </transitionSet>

</transitionSet>
```

**Issue**

The "edit_text_container" jumps up before the transition starts. I have tested it on JB, KK, LP. It works without jump only on JB. Maybe it was because of the small screen size, I am not sure.

**Comment by ealymbaev** [comment]
I have also tried using native transitions in Lollipop with the same layouts and transition xml's. It works correctly there.

**Comment by andkulikov** [comment]
fixed  in 1.3.2
